packages/elements/src/components/app/app.component.ts

```diff
@@ -27,6 +27,9 @@ interface TestContext extends BaseContext {
   result?: MetricsResult<TestFileModel, TestMetrics>;
 }
 
+// How many milliseconds should be between updates
+const UPDATE_CYCLE_TIME = 100;
```

Comment:
I think it would be useful to express this (as a comment) in the form of given x amount of mutants which currently takes y time to render (on 'a random' computer) so we can get a feeling of the performance characteristics. 

Like, given 5000 mutants on a single page that take 50ms to render we would have a margin of 50 percent between updates. Something like that, so we can judge if the cycle time should ever be changed up or down.

Reply:
Is this better?

Reply:
Yes but I would still like to see a real world example of the time it actually takes for 1 render of x amount of mutants to take place so we can put the cycle time number into context.

Reply:
I'm not sure if I can measure that, but rendering is probably instantaneous?

Previously there were 6k+ calls to the `update` method, now there are `3` (https://profiler.firefox.com/public/fftmqbk76rwn26g2p9wwjzwwr6ef7b6gqh052g8/calltree/?globalTrackOrder=0w3&implementation=js&range=4862m780&search=updateAllMe&thread=6&v=9)

Reply:
Those three updates took 6ms, so 2ms for one update? (on my laptop)